### ISSUE TITLE
demo components won't compile - DateTime double timezone specification

### ISSUE THREAD

**Issue by Peeripapo** [vdm-io/Joomla-Component-Builder#39]
Hi, I can't seem to compile the demo components. Keep getting the same error on compile for all 3 components:

DateTime::__construct(): Failed to parse time string (-001-11-29T23:00:00+00:00) at position 7 (-): Double timezone specification

I'm using a fresh 3.6.5 joomla install and CB 2.3.0

**Comment by Llewellynvdm** [comment]
Hi this is a bug, and I am working on it... 

To get around it open all views and fields and save them (make sure the created and modified dates are set), the error is due to these views and fields not having created, and modified dates.

Should have this fixed in a few days....

**Comment by Peeripapo** [comment]
So here's what I've done. I ran this sql code on phpmyadmin and that fixed it. I guessed it would be faster than editing the fields and views one by one and it worked :)

update cb_componentbuilder_custom_admin_view SET created='2017-01-01 08:00:00', modified='2017-01-01 08:00:00' WHERE 1
update cb_componentbuilder_admin_view SET created='2017-01-01 08:00:00', modified='2017-01-01 08:00:00' WHERE 1
update cb_componentbuilder_site_view SET created='2017-01-01 08:00:00', modified='2017-01-01 08:00:00' WHERE 1
update cb_componentbuilder_field SET created='2017-01-01 08:00:00', modified='2017-01-01 08:00:00' WHERE 1

Just remember to replace the prefix with the right one for your database and this should do the trick.
